src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.html

```diff
@@ -43,6 +43,23 @@
 
 <div class="edit-sqi" *ngIf="exercise.gradingInstructionFeedbackUsed">
     <div class="grading-instructions-update-border">
+        <div class="instructions" style="padding-bottom: 12px">
```

Comment:
I think we should stick to bootstrap classes, as we do throughout the project instead of style padding.
Here is a link for reference: https://getbootstrap.com/docs/5.0/utilities/spacing/

Adding a class like `pb-2` maybe would do the trick

Reply:
coool, good to know 👍